Source/WebCore/Modules/paymentrequest/PaymentRequestUpdateEvent.h

```diff
@@ -40,16 +40,16 @@ class PaymentRequestUpdateEvent : public Event {
 public:
     template <typename... Args> static Ref<PaymentRequestUpdateEvent> create(Args&&... args)
     {
-        return adoptRef(*new PaymentRequestUpdateEvent(std::forward<Args>(args)...));
+        return adoptRef(*new PaymentRequestUpdateEvent(EventInterfaceType::PaymentRequestUpdateEvent, std::forward<Args>(args)...));
     }
     ~PaymentRequestUpdateEvent();
     ExceptionOr<void> updateWith(Ref<DOMPromise>&&);
 
     bool didCallUpdateWith() const { return m_waitForUpdate; }
 
 protected:
-    explicit PaymentRequestUpdateEvent(const AtomString& type);
-    PaymentRequestUpdateEvent(const AtomString& type, const PaymentRequestUpdateEventInit&);
+    explicit PaymentRequestUpdateEvent(enum EventInterfaceType, const AtomString& type);
```

Comment:
Surprised with need `enum` here?

Reply:
Yeah, it's because `EventInterface` enum has a value named `EventInterfaceType`. My plan to get rid of this enum completely once we migrate WebKitAdditions to new code.